### ISSUE TITLE
Need for help 

### ISSUE THREAD

**Issue by Azurz** [derklaro/ps-system#62]
Hello I followed the tuto I find myself with that how I do after create the npc because the commands in game I need to see screen ?
thank you

![image](https://user-images.githubusercontent.com/35930914/151667263-6a937daa-3f13-4682-ab09-5cdbe8b0e7ad.png)
![image](https://user-images.githubusercontent.com/35930914/151667382-51f66574-92dd-4c9f-bcdb-602ebd744eb9.png)

I managed to create an npc but it does not save?
But I don't understand what I did wrong 
![image](https://user-images.githubusercontent.com/35930914/151671937-43881c52-0b30-4b87-9c5b-f2ecb2fec005.png)

**Comment by ghost** [comment]
Du musst /createnpc machen

**Comment by Azurz** [comment]
Yep I succeeded thanks mets I have the impression that it does not take well the template request and that it takes only the Global server

**Comment by Azurz** [comment]
And the npgs are not well saved too

**Comment by ghost** [comment]
Can you help me in my issue ? https://github.com/derklaro/ps-system/issues/80

**Comment by ghost** [comment]
> 

Are they not in the config?
